synchdb.c

```diff
@@ -2046,6 +2046,35 @@ dbz_mark_batch_complete(int batchid)
 	return 0;
 }
 
+static void
+remove_dbz_metadata_files(const char * name)
+{
+	struct dirent *entry;
+	char filepath[256] = {0};
```

Comment:
Using a defined constant instead of 256?
for example #define SYNCHDB_PATH_MAX 1024